enzyme/mkv.py

```diff
@@ -158,13 +159,20 @@ def fromelement(cls, element):
         :type element: :class:`~enzyme.parsers.ebml.Element`
 
         """
+        uid = binascii.hexlify(element.get('SegmentUID').getvalue()).decode().upper()
         title = element.get('Title')
         duration = element.get('Duration')
         date_utc = element.get('DateUTC')
         timecode_scale = element.get('TimecodeScale')
         muxing_app = element.get('MuxingApp')
         writing_app = element.get('WritingApp')
-        return cls(title, duration, date_utc, timecode_scale, muxing_app, writing_app)
+        return cls(uid, title, duration, date_utc, timecode_scale, muxing_app, writing_app)
+
+    def to_dict(self):
+        info_dict = self.__dict__.copy()
+        info_dict['duration'] = self.duration.seconds + self.duration.microseconds/1e6
```

Comment:
Is `self.duration.microseconds/1e6` pep8-valid? I think it needs extra spaces around the `/`